src/index.js

```diff
@@ -330,7 +330,11 @@ stdlib.values = function(val, path){
   return _.values(val);
 };
 stdlib.intersection = _.intersection;
-stdlib.union = _.union;
+stdlib.union = function() {
+  var args = Array.prototype.slice.call(arguments);
+  args.push(_.isEqual);
+  return _.unionWith.apply(_, args);
```

Comment:
- Get the function's arguments in an array.
- Add the comparator function `_.isEqual` to the end of the arguments list for `_.unionWith`.
- Call `_.unionWith`, passing in the modified arguments with the comparator function.